MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/DataProcessorUI/QDataProcessorWidget.h

```diff
@@ -86,7 +86,7 @@ class EXPORT_OPT_MANTIDQT_MANTIDWIDGETS QDataProcessorWidget
                             const std::string &defaultValue) override;
   bool askUserYesNo(std::string prompt, std::string title) override;
   void giveUserWarning(std::string prompt, std::string title) override;
-  void giveUserCritical(std::string prompt, std::string title) override;
+  void giveUserCritical(QString prompt, std::string title) override;
```

Comment:
I agree. I think the idea of using other toolkits is nice but it is not really that practical.

Edward, can you make the changes?